invoice/src/main/java/org/killbill/billing/invoice/template/formatters/DefaultInvoiceFormatter.java

```diff
@@ -213,20 +226,35 @@ public BigDecimal getBalance() {
 
     @Override
     public String getFormattedChargedAmount() {
-        final NumberFormat number = NumberFormat.getCurrencyInstance(locale);
-        return number.format(getChargedAmount().doubleValue());
+        return getFormattedAmountByLocaleAndInvoiceCurrency(getChargedAmount());
     }
 
     @Override
     public String getFormattedPaidAmount() {
-        final NumberFormat number = NumberFormat.getCurrencyInstance(locale);
-        return number.format(getPaidAmount().doubleValue());
+        return getFormattedAmountByLocaleAndInvoiceCurrency(getPaidAmount());
     }
 
     @Override
     public String getFormattedBalance() {
-        final NumberFormat number = NumberFormat.getCurrencyInstance(locale);
-        return number.format(getBalance().doubleValue());
+        return getFormattedAmountByLocaleAndInvoiceCurrency(getBalance());
+    }
+
+    /**
+     * Returns the amount with the correct currency symbol
+     *
+     * @param amount
+     * @return
```

Comment:
Either document the fields or simply put a non-Javadoc comment, e.g.:

```
// Returns the amount with the correct currency symbol
```

Otherwise, the build will fail on Java 8 where doclint is [enabled by default](http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html).